p4-16/psa/psa.p4

```diff
@@ -577,7 +577,7 @@ extern ActionSelector {
 
 // BEGIN:Digest_extern
 extern Digest<T> {
-  Digest(PortId_t receiver);         /// define a digest stream to receiver
+  Digest();                       /// define a digest stream to receiver
```

Comment:
'receiver' in comment no longer refers to anything in the code.  Maybe change it to 'define a digest stream to the control plane' ?

Reply:
@hanw I'd be happy to approve these changes and merge them in, if you can update the comment to match the code changes.  Thanks.